src/Sleep.js

```diff
@@ -17,42 +17,24 @@ class Sleep {
     return [...specificUser].splice(-7 * week, 7).map(day => day.date);
   }
 
-  returnAvgSleepHours() {
+  returnAvgSleepInfo(sleepInfo) {
     let specificUser = this.findUser();
-    return Number((specificUser.reduce((totalHours, day) => {
-      totalHours += day.hoursSlept;
-      return totalHours;
+    return Number((specificUser.reduce((acc, day) => {
+      acc += day[sleepInfo];
+      return acc;
```

Comment:
This all looks good Dustin - this is a nice use of bracket notation coupled with using a parameter - it's crazy how much less code this creates! good refactoring!